test/modules-option.test.js

```diff
@@ -1566,6 +1566,7 @@ describe('"modules" option', () => {
   it('should work with the "namedExport" option', async () => {
     const compiler = getCompiler("./modules/namedExport/base/index.js", {
       modules: {
+        exportLocalsConvention: "camelCaseOnly",
```

Comment:
Why it was added? Sounds like a breaking change

Reply:
I mentioned this in the comment above. The changes didn't have any effect on the unit test. I will remove.